docker/scripts/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir -p /srv/montag/books/{file_store,import_watch}
+mkdir -p /srv/montag/metadata/{db,config,logs}
```

Comment:
Readme.MD gives this example:
`  -v /var/lib/montag/data:/srv/montag/data` 
Is it possible that line 4 here should read 'mkdir -p /srv/montag/data/{db,config,logs}' ?

Reply:
Alternatively 'metadata' was intended and the readme should be fixed, it seems 'metadata' is used in several places and appears to be the better term.

Reply:
changed to metadata in readme